specification/network/resource-manager/Microsoft.Network/stable/2021-05-01/firewallPolicy.json

```diff
@@ -567,6 +567,256 @@
           "nextLinkName": "nextLink"
         }
       }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Network/firewallPolicies/{firewallPolicyName}/listIdpsSignatures": {
+      "post": {
+        "operationId": "FirewallPolicyIdpsSignatures_List",
+        "description": "Retrieves the current status of IDPS signatures for the relevant policy",
+        "parameters": [
+          {
+            "name": "resourceGroupName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the resource group."
+          },
+          {
+            "name": "firewallPolicyName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the Firewall Policy."
+          },
+          {
+            "$ref": "./network.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "name": "parameters",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/IDPSQueryObject"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success. The operation returns an IDPS Query Response",
+            "schema": {
```

Comment:
>schema [](http://example.com/codeflow?start=13&length=6)

Consider following the x-ms-pageable pattern defined here: [https://github.com/Azure/autorest/blob/main/docs/extensions/readme.md#x-ms-pageable](https://github.com/Azure/autorest/blob/main/docs/extensions/readme.md#x-ms-pageable). Although the SDKs use it for GET methods, it would improve API consistency to use the same pattern.